pkgs/by-name/et/ethercat/common.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  fetchFromGitLab,
+  stdenv,
+  gitUpdater,
+  autoreconfHook,
+  pkg-config,
+  withDocs ? false,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ethercat";
+  version = "1.6.1";
+
+  src = fetchFromGitLab {
+    owner = "etherlab.org";
+    repo = "ethercat";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-Ki8WUKUW4O9SNFTVIXrWpL6suCu7j2nLQtpWVsUoiK0=";
+    fetchSubmodules = withDocs;
+    leaveDotGit = withDocs;
```

Comment:
This will not work as the hash changes

Reply:
I've reworked this in da57d7998d6f3b3b8e009f53050f062ad77bf291